tests/test_views.py

```diff
@@ -325,6 +324,7 @@ def test_base_context_values(self, client):
             assert str(response.context[key]) == str(expected_context[key])
 
         assert list(response.context['related_url_list']) == related_urls
+        assert response.context['url_data'] == views.create_url_list(project, related_urls + [url])
```

Comment:
I don't follow why this was pulled out of our `expected_context` and just validated in the loop.

Reply:
I think it no longer casts to str properly. I remember looking at that too.

Reply:
yes, `url_data` is not converting to string data type which is why i brought it out of the loop.

Reply:
ah ok, good enough for me